.github/workflows/ado-migration-process-workitem-backfill.yml

```diff
@@ -83,6 +83,7 @@ jobs:
         shell: pwsh
         run: |
           Get-ChildItem "${{ vars.WorkItemMigratorDirectory }}/logs" -Recurse | Remove-Item -Recurse
+          Get-ChildItem "c:\\temp\\WorkItemAttachmentWorkingFolder" -Recurse | Remove-Item -Recurse
```

Comment:
Don't hardcode temp path. Use $env:TEMP or similar. Or use actions working folder to make more generic.